docs/tools/vdb_table/data/pinecone.json

```diff
@@ -68,38 +68,38 @@
     "comment": ""
   },
   "embeddings_text": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
```

Comment:
This column is to indicate that the embeddings can be computed within the DB.
An external library maintained by the DB vendor can count as "partial" support

Reply:
noted and updated